app/components/error_list/connection_error.js

```diff
@@ -17,12 +18,19 @@ const style = StyleSheet.create({
     }
 });
 
-function ConnectionError() {
+function ConnectionError(intl) {
     return (
         <View style={style.container}>
-            <Text>{'I am a general error'}</Text>
+            <Text>
+                {
+                  intl.formatMessage({
```

Comment:
you don't need to add `intl` here if you use `<FormattedText>` component found in `app/components/formatted_text`